tests/behat/features/profile.feature

```diff
@@ -48,16 +48,16 @@ Feature: Manage my own settings
     And I fill in "New Password" with "a"
     And I fill in "Confirm Password" with "a"
     And I press the "Save" button
-    Then I should see "Password is too short"
+    Then I should see "Please increase password strength by adding some of the following characters: uppercase, digits, punctuation"
```

Comment:
Required changes after this PR : https://github.com/silverstripe/silverstripe-framework/pull/10966

Reply:
Test fails since we don't have any setting for `password strength`.